src/bin/version.rs

```diff
@@ -3,8 +3,8 @@ use std::env;
 use cargo;
 use cargo::util::{CliResult, Config};
 
-#[derive(RustcDecodable)]
-pub struct Options;
+#[derive(Deserialize)]
+pub struct Options {}
```

Comment:
Docopt currently doesn't support unit structs, as a workaround I changed it to empty struct.